src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Symbology/UseDistanceCompositeSym/UseDistanceCompositeSym.xaml.cs

```diff
@@ -7,89 +7,99 @@
 // "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific 
 // language governing permissions and limitations under the License.
 
+using ArcGISRuntime.Samples.Managers;
 using Esri.ArcGISRuntime.Geometry;
 using Esri.ArcGISRuntime.Mapping;
 using Esri.ArcGISRuntime.Symbology;
 using Esri.ArcGISRuntime.UI;
 using System;
 using System.Drawing;
+using System.Threading.Tasks;
 
 namespace ArcGISRuntime.WPF.Samples.UseDistanceCompositeSym
 {
     [ArcGISRuntime.Samples.Shared.Attributes.Sample(
         "Distance composite symbol",
         "Symbology",
         "This sample demonstrates how to create a `DistanceCompositeSceneSymbol` with unique marker symbols to display at various distances from the camera.",
-        "1. When the Scene initially loads, you will be looking at a point graphic symbolized with a 3D cone symbol.\n2. Zoom slightly closer to the graphic and the symbol will change to a blue 3D cube symbol.\n3. Zoom farther from the graphic and the symbol will change back to a 3D cone and then to a simple marker symbol.")]
-
+        "1. When the Scene initially loads, you will be looking at a point graphic symbolized with a 3D model symbol (airplane).\n2. Zoom away slightly from the graphic and the symbol will change to a blue 3D cone symbol (pointing in the direction of flight).\n3. Zoom farther from the graphic and the symbol will change to a blue circle marker symbol.")]
     public partial class UseDistanceCompositeSym
     {
-        // URL for an image service to use as an elevation source
-        private string _elevationSourceUrl = @"https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer";
-
         public UseDistanceCompositeSym()
         {
             InitializeComponent();
 
-            // Create the Scene, basemap, graphic, and composite symbol
+            // Create the Scene, basemap, graphic, and composite symbol.
             Initialize();
         }
 
-        private void Initialize()
+        private async Task Initialize()
```

Comment:
The sample is generating a compile warning because of the Initialize signature. Probably best to modify to the following (I learned this trick from reviewing the LineOfSightGeoElement.xaml.cs file):

private async void Initialize()